packages/tx/src/eip2930Transaction.ts

```diff
@@ -158,6 +160,11 @@ export class AccessListEIP2930Transaction extends BaseTransaction<AccessListEIP2
       gasPrice: this.gasPrice,
     })
 
+    const notArray = { ...txData }
```

Comment:
TODO we copy this, how to optimize?

Reply:
@acolytec3 we can now just dump in txData into the static func right?

Reply:
Should be able to. That's what the tests are doing and why since were failing since we were just blindly checking everything that was passed in

Reply:
hmmm seems like you are talking the same thing here, what issues are you facing if you just use txData in the validation call?